packages/@lwc/ssr-runtime/src/reflection.ts

```diff
@@ -16,16 +17,85 @@ import {
 
 import type { LightningElement } from './lightning-element';
 
-// Eventually include globals that also reflect
-const attrsToProps = AriaAttrNameToPropNameMap;
+/**
+ * Map of global attribute or ARIA attribute to the corresponding property name.
+ * Not all global attributes are included, just those from `HTMLElementTheGoodParts`.
+ */
+const attrsToProps = assign(create(null), {
+    accesskey: 'accessKey',
+    dir: 'dir',
+    draggable: 'draggable',
+    hidden: 'hidden',
+    id: 'id',
+    lang: 'lang',
+    spellcheck: 'spellcheck',
+    tabindex: 'tabIndex',
+    title: 'title',
```

Comment:
You can already find the [list of global attributes](https://github.com/salesforce/lwc/blob/75ff766c8dabdd28e3308e3f2c8f763c385b056a/packages/%40lwc/shared/src/html-attributes.ts#L61-L90) and [irregular attr-prop name mappings](https://github.com/salesforce/lwc/blob/75ff766c8dabdd28e3308e3f2c8f763c385b056a/packages/%40lwc/shared/src/html-attributes.ts#L102-L117) in `@lwc/shared`.

Reply:
This list is distinct from both; it is the attr-prop mapping for the subset of global attributes that are used by `HTMLElementTheGoodParts`. We need a static list of either included or excluded attributes; using the includes is simpler because we don't need to filter. And the mapping you referenced is a prop -> attr mapping, but we need attr -> prop, so that would also take additional parsing. Technically we could, but this seemed simpler.